src/tools/sssctl/sssctl_config.c

```diff
@@ -64,6 +64,9 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
     /* Open config file */
     ret = sss_ini_config_file_open(init_data, SSSD_CONFIG_FILE);
     if (ret != EOK) {
+        if (ret == ENOENT) {
```

Comment:
IMHO second DEBUG message is unnecessary in this case. It does not improve anything.

BTW Should we mention patch to sssd.conf in ERROR message? `SSSD_CONFIG_FILE`